### PR TITLE
added new fences to wirecutter

### LINKED ISSUE #6420: Fences, that do not work with the wire cutter
**Arma 3 Version:** `1.82` (stable)
**CBA Version:** `3.7.1` (stable)
**ACE3 Version:** `3.12.2` (stable)

**Mods:**
```
- CBA_A3
- ace
- australia map
```

**Description:**
- Some fences from Vanilla and Apex-DLC can't be cut with the wire cutter. And fences on the australian map, which use the same 3d-model of vanilla can't be cut, as well.

here the 3d models + classnames (Vanilla):
netfence_01_m_4m_f.p3d "Land_NetFence_01_m_4m_F"
netfence_01_m_8m_f.p3d "Land_NetFence_01_m_8m_F"
netfence_02_m_2m_f.p3d "Land_NetFence_02_m_2m_F"
netfence_02_m_4m_f.p3d "Land_NetFence_02_m_4m_F"
netfence_02_m_8m_f.p3d "Land_NetFence_02_m_8m_F"
wiredfence_01_4m_f.p3d "Land_WiredFence_01_4m_F"
wiredfence_01_8m_f.p3d "Land_WiredFence_01_8m_F"
wiredfence_01_16m_f.p3d "Land_WiredFence_01_16m_F"
slums01_8m.p3d "Land_Slums01_8m"

(Australia-Map):
indfnc_3_f.p3d "xcam_IndFnc_3_F"
indfnc_3_hole_f.p3d "xcam_IndFnc_3_Hole_F"
indfnc_9_f.p3d "xcam_IndFnc_9_F"
indfnc_corner_f.p3d "xcam_IndFnc_Corner_F"
mil_wiredfence_f.p3d "xcam_Mil_WiredFence_F"
net_fence_4m_f.p3d "xcam_Net_Fence_4m_F"
net_fence_8m_f.p3d "xcam_Net_Fence_8m_F"
new_wiredfence_5m_f.p3d "xcam_New_WiredFence_5m_F"
new_wiredfence_10m_f.p3d "xcam_New_WiredFence_10m_F"
slums01_8m.p3d "xcam_Slums01_8m"
sportground_fence_f.p3d "xcam_SportGround_fence_F"
sportground_fence_nolc_f.p3d "xcam_SportGround_fence_noLC_F"
plot_provizorni.p3d "WireFence"
plp_ctm_partitioningfencegrey.p3d "plp_ctm_PartitioningFenceGrey"
fence.p3d "Land_Fence"

### PR DESCRIPTION
**Pull request by shukari**
**When merged this pull request will:**
- added new isFence classes
- added all isFence model p3ds and special xcam dences to macro FENCE_P3DS

resolve #6420

### CODE REVIEW

**Comment by jonpas** [comment]
Use tabs, as per our coding guidelines.

**Comment by shukari** [comment]
@jonpas should i use tab... the error say i should not... confused

**Comment by jonpas** [comment]
No, you should not. You should use spaces.

**Comment by jonpas** [comment]
Oh, I just realized haha, I wrote it the other way around, sorry!

**Comment by shukari** [comment]
@jonpas ok, is fixed

**Comment by PabstMirror** [comment]
Fixed merge conflicts from #6389 
I think I got the correct union of both lists, let me know if I missed anything.

File: addons/logistics_wirecutter/script_component.hpp [hunk 1]
```diff
@@ -16,28 +16,42 @@
 
 #include "\z\ace\addons\main\script_macros.hpp"
 
+
```
Comment (mharis001) [inline addons/logistics_wirecutter/script_component.hpp]: Not needed extra line

**Pull request merged**